client/app/scripts/liveblog-edit/views/filter-by-member.html

```diff
@@ -0,0 +1,41 @@
+<button ng-click="vm.toggleSelector()" class="btn btn-info btn-mini btn--plus"><i class="icon-white icon-plus-large"></i></button>
+<div class="dropdown-content" ng-click="$event.stopPropagation();" ng-if="vm.openSelector">
+    <div class="user-select">
+        <div>
+            <form>
+                <label for="assignee-search" translate>Select users</label>
+                <input type="text" name="assignee-search" ng-model="vm.search">
+            </form>
+            <div class="search-result">
+                <ul class="users-list-embed">
+                    <li ng-repeat="user in vm.members | filter:vm.search | orderBy: 'display_name'"
+                        ng-click="vm.toggleUserInPreselection(user)"
+                        ng-class="{'active': vm.isUserInPreselection(user)}"
+                        data-username="{{user.username}}">
+                        <figure class="avatar small"><img sd-user-avatar data-src="user.picture_url"></figure>
+                        <div class="name">{{user.display_name}}</div>
```

Comment:
Can this change in the scope's lifetime? If not, it is recommended to use one-time binding to reduce the total number of $watches.